### PR TITLE
new-dashboard new-navbars

### PR DESCRIPTION
**Pull request by jacekkolasa**
this PR is based on #8363, just with different, alpha version of new dashboard with new navbars/iframes

not all features are included here, but at least sso is working and spaces-bar, rooms, streamed and visited nodes are shown. That's because some stuff is not yet reviewed/merged
I'll update this PR tomorrow

To use this, cloud base url should be set to test environment url

### CODE REVIEW

File: packaging/dashboard.version [hunk 1]
```diff
@@ -0,0 +1 @@
+v1.0-alpha.4
```
Comment (ilyam8) [inline packaging/dashboard.version]: why it is different from this
https://github.com/netdata/netdata/pull/8363/files?file-filters%5B%5D=.am&file-filters%5B%5D=.checksums&file-filters%5B%5D=.in&file-filters%5B%5D=.sh&file-filters%5B%5D=.version#diff-15eb7dcc98aba18503e2174a75a5b21b

Comment (ilyam8) [reply packaging/dashboard.version]: ok, i see this one is the latest on was released 31 minutes ago, #8363 is not updated

**Comment by thiagoftsm** [comment]
When I tried to see the alarms on dashboard in my local computer, I am seeing this error:

err20

**Comment by jacekkolasa** [comment]
@thiagoftsm possibly again problem with cache? Have you tried private mode or different browser? Do you remember if you saw something in console at that time? I cannot recreate this bug :(

**Comment by thiagoftsm** [comment]
I am retesting now on private mode, and I got the following results:

- You are right, I had problem with cache. I apologize for this, but yesterday I had to test different features and I did not use `private window` during this specific test.
- This time I used the button that is inside the `iframe` that normally show hosts instead the `Sign in` button, when I finish the login, it did not return for the original dashboard, it went direct to the cloud, this is a behavior change, is this expected?
- When I selected my slave, instead it goes to the bigger `iframe` it continues overwritten the slaves I have.

**Comment by jacekkolasa** [comment]
thanks @thiagoftsm for checking that

> This time I used the button that is inside the iframe that normally show hosts instead the Sign in button, when I finish the login, it did not return for the original dashboard, it went direct to the cloud, this is a behavior change, is this expected?

Yes, that's expected. I can change that if @manos-saratsis or @rduffield thinks it's worth it.


> When I selected my slave, instead it goes to the bigger iframe it continues overwritten the slaves I have.

I'll fix that in the cloud-iframes directly, thanks Thiago

**Comment by jacekkolasa** [comment]
closing in favor of https://github.com/netdata/netdata/pull/8818

**Pull request closed without merging**